webpack.common.js

```diff
@@ -4,13 +4,13 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 
 module.exports = {
-  entry: './src/index.js',
+  entry: './src/app/index.js',
```

Comment:
I'm just wondering why we need this extra folder? 🤔

Reply:
I used it according to react components video, but I think, the structure doesn`t include src folder, so I deleted app folder.